src/components/mod.rs

```diff
@@ -6,14 +6,97 @@ mod diff;
 mod download;
 
 #[derive(Debug, Clone, interactive_clap::InteractiveClap)]
-#[interactive_clap(context = near_cli_rs::GlobalContext)]
+#[interactive_clap(input_context = near_cli_rs::GlobalContext)]
+#[interactive_clap(output_context = ComponentsContext)]
+#[interactive_clap(skip_default_from_cli)]
 pub struct Components {
+    /// Change SocialDb prefix
+    #[interactive_clap(long)]
+    #[interactive_clap(skip_default_input_arg)]
+    social_db_prefix: Option<String>,
     #[interactive_clap(subcommand)]
     command: self::ComponentsCommand,
 }
 
+#[derive(Clone)]
+pub struct ComponentsContext {
+    pub global_context: near_cli_rs::GlobalContext,
+    pub social_db_prefix: String,
+}
+
+impl ComponentsContext {
+    pub fn from_previous_context(
+        previous_context: near_cli_rs::GlobalContext,
+        scope: &<Components as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        Ok(Self {
+            global_context: previous_context,
+            social_db_prefix: scope
+                .social_db_prefix
+                .clone()
+                .unwrap_or("widget".to_owned()),
+        })
+    }
+}
+
+impl interactive_clap::FromCli for Components {
+    type FromCliContext = near_cli_rs::GlobalContext;
+    type FromCliError = color_eyre::eyre::Error;
+    fn from_cli(
+        optional_clap_variant: Option<<Self as interactive_clap::ToCli>::CliVariant>,
+        context: Self::FromCliContext,
+    ) -> interactive_clap::ResultFromCli<
+        <Self as interactive_clap::ToCli>::CliVariant,
+        Self::FromCliError,
+    >
+    where
+        Self: Sized + interactive_clap::ToCli,
+    {
+        let mut clap_variant = optional_clap_variant.unwrap_or_default();
+
+        if clap_variant.social_db_prefix.is_none() {
+            clap_variant.social_db_prefix = match Self::input_social_db_prefix(&context) {
+                Ok(optional_social_db_prefix) => optional_social_db_prefix,
+                Err(err) => return interactive_clap::ResultFromCli::Err(Some(clap_variant), err),
+            };
+        }
+        let social_db_prefix = clap_variant.social_db_prefix.clone();
```

Comment:
In interactive-clap, when `skip_interactive_input` is used for the field, it should just generate the following code for `from_cli` method:

```suggestion
        let social_db_prefix = clap_variant.social_db_prefix.clone();
```